base/common/src/main/java/com/netscape/certsrv/base/PKIException.java

```diff
@@ -52,32 +52,20 @@ public PKIException(int code, String message, Throwable cause) {
         this.code = code;
     }
 
-    public PKIException(Response.Status status) {
-        this(status, status.getReasonPhrase(), null);
-    }
-
     public PKIException(String message) {
-        this(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), message, null);
+        this(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, message, null);
     }
 
     public PKIException(int code, String message) {
         this(code, message, null);
     }
 
-    public PKIException(Response.Status status, String message) {
-        this(status.getStatusCode(), message, null);
-    }
-
     public PKIException(Throwable cause) {
-        this(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), cause.getMessage(), cause);
+        this(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Internal Server Error", cause);
```

Comment:
Could we include `cause.getMessage()` in the exception message to help troubleshooting?

Reply:
The message is printed to the end-user so it should not provide internal details and there are are logs including the messages for the majority of (if not all) cases to help troubleshooting. However, since this was the original behaviour I am reverting this message and we can evaluate later if it is OK to have.